### PR TITLE
wildfly-as 10.1.0.Final

### PR DESCRIPTION
**Pull request by rhusar**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/wildfly-as.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ def install
   end
 
   def caveats; <<-EOS.undent
-    The home of WildFly Application Server 10 is:
+    The home of WildFly Application Server 10.1 is:
```
Comment (zmwangx) [inline Formula/wildfly-as.rb]: Is this change really necessary?

Comment (rhusar) [inline Formula/wildfly-as.rb]: No, if you feel like its better to not do this change, let me know, i ll revert it.

Comment (zmwangx) [inline Formula/wildfly-as.rb]: It's fine to just keep it IMO.

Comment (rhusar) [inline Formula/wildfly-as.rb]: Ok, done.

**Comment by zmwangx** [comment]
Merged as 97806e9. Thank you for your contribution to Homebrew!
